src/test/resources/com/jcabi/w3c/invalid-html-response.xml-without-schema

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
@prondzyn Check this: https://github.com/teamed/qulice/blob/master/qulice-checkstyle/pom.xml#L112.